roles/ceph-rgw/defaults/main.yml

```diff
@@ -21,6 +21,28 @@ cephx: true
 rgw_pull_port: "{{ radosgw_civetweb_port }}"
 rgw_pull_proto: "http"
 
+########
+#TUNING#
+########
+# The below values were determined to be safe defaults to 
+# prevent users from putting too many objects into their
+# buckets which causes performance and stability issues.
+#ceph_conf_overrides:
+#    rgw:
+#      radosgw_civetweb_num_threads: 50
```

Comment:
@mattbenjamin is the suggested number for this supposed to be lower? This is what it defaults to now?